server/src/main/java/org/elasticsearch/common/util/CuckooFilter.java

```diff
@@ -507,4 +526,205 @@ public boolean equals(Object other) {
             && Objects.equals(this.count, that.count)
             && Objects.equals(this.evictedFingerprint, that.evictedFingerprint);
     }
+
+    //  Forked from Lucene's Packed64 class.vThe main difference is that this version
```

Comment:
s/.vThe/. The/

Maybe this should just be javadoc? No big deal either way.

Reply:
done